listenbrainz_spark/mlhd/popularity.py

```diff
@@ -0,0 +1,95 @@
+from more_itertools import chunked
+
+from listenbrainz_spark import config
+from listenbrainz_spark.path import MLHD_PLUS_DATA_DIRECTORY
+from listenbrainz_spark.stats import run_query
+
+
+STATS_PER_MESSAGE = 10000
+
+
+def generate(name, query):
```

Comment:
this name is a bit too generic. generate_popularity_stats?

Reply:
fixed